securesubmit/services/gateway.py

```diff
@@ -34,11 +34,13 @@ class HpsSoapGatewayService(object):
     _base_config = None
     _url = None
     _logging = False
+    _timeout = None
 
-    def __init__(self, config=None, enable_logging=False):
+    def __init__(self, config=None, enable_logging=False, timeout=None):
```

Comment:
As opposed to adding the timeout as an additional parameter to the gateway objects I would add it to the service configuration object which is already passed in.